test/utils/wabt_engine.cpp

```diff
@@ -8,17 +8,23 @@
 
 #include <test/utils/adler32.hpp>
 #include <test/utils/wasm_engine.hpp>
+#include <algorithm>
 #include <cassert>
 
 namespace fizzy::test
 {
 class WabtEngine final : public WasmEngine
 {
-    wabt::interp::Environment m_env;
-    wabt::interp::DefinedModule* m_module{nullptr};
-
-    // WABT Executor/Thread with default options.
-    wabt::interp::Executor m_executor{&m_env};
+    // mutable because getting RefPtr from Ref requires non-const Store
+    mutable wabt::interp::Store m_store{[]() noexcept {
```

Comment:
Lol when adding `constexpr` here clang-format gets crazy
```suggestion
    mutable wabt::interp::Store m_store{[]() constexpr noexcept {
```

Reply:
Good to report bug.

Reply:
Additional pair of parentheses helped.